dev/NavigationView/NavigationViewItemAutomationPeer.cpp

```diff
@@ -47,10 +47,10 @@ winrt::IInspectable NavigationViewItemAutomationPeer::GetPatternCore(winrt::Patt
 {
     winrt::IInspectable result = __super::GetPatternCore(pattern);
 
-    if (!result && pattern == winrt::PatternInterface::Invoke)
+    if (!result && (pattern == winrt::PatternInterface::SelectionItem ||
```

Comment:
>result [](start = 9, length = 6)

Is it weird that we default to the supers implementation of selectionItem and Invoke?